JuiceMaker/JuiceMaker/Controller/ViewController.swift

```diff
@@ -7,12 +7,71 @@
 import UIKit
 
 class ViewController: UIViewController {
-
+    var juiceMaker = JuiceMaker()
+    
+    @IBOutlet weak var strawberryStock: UILabel!
+    @IBOutlet weak var bananaStock: UILabel!
+    @IBOutlet weak var pineappleStock: UILabel!
+    @IBOutlet weak var kiwiStock: UILabel!
+    @IBOutlet weak var mangoStock: UILabel!
+    
     override func viewDidLoad() {
         super.viewDidLoad()
-        // Do any additional setup after loading the view.
+        updateStock()
+    }
+    
+    @IBAction func tabEditStock(_ sender: UIBarButtonItem) {
+        self.changeView()
+    }
+    
+    @IBAction func order(_ sender: UIButton) {
+        guard let juice = Menu(rawValue: sender.tag) else {
+            return
+        }
+        do {
+            try juiceMaker.make(menu: juice)
+        } catch {
+            showlackOfStockAlert()
+        }
+        updateStock()
+        showCompletionAlert(menu: juice)
+    }
+    
+    func updateStock() {
+        strawberryStock.text = juiceMaker.store.notifyStock(fruit: .strawberry).description
```

Comment:
notifyStock이 Int를 뱉는데 결국 필요한 값은 String 이니깐 description을 사용하시는 것 같은데, 아래처럼 직접적으로 변환하는게 좋을 것 같아요. 
```suggestion
        strawberryStock.text = "\(juiceMaker.store.notifyStock(fruit: .strawberry))"
```
왜냐하면 description은 Int 말고도 다른 어떤 타입도 가지고 있는 값이라 description을 쓰면 어떤 텍스트 값인지 바로 보이지 않지만, 위처럼 쓰면 더 분명하게 text값으로 어떤값이 할당되는지 잘 보여서입니다.

description과 관련된 Customstringconvertible protocol도 알고 계시면 좋습니다.
https://developer.apple.com/documentation/swift/customstringconvertible

Reply:
description을 쓸 때보다 문자열 보간법처럼 사용해주니까 확실히 Int-> String 타입으로 변환해주고자 하는 목적이 확실히 보이네요! 변경해주었습니다 🚀